src/components/views/admin/BuilderView.tsx

```diff
@@ -50,7 +45,16 @@ const BuilderView: FC = () => {
   const filteredObjects = groupedObjects.filter((group) =>
     group.objects.some((obj) => obj.type === selectedType),
   );
-
+  const [ratings, setRatings] = useState<{ [key: string]: number | null }>({});
+  const dimensions = [
+    { value: 'mock_type', label: 'Mock Info' },
```

Comment:
Do you plan on keeping this ?

Reply:
No, mock_type is just for showing a simple result, so I would know how it looks like.